CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### To be Released
 
+* fix(warning-data-stream): change data stream on which warning is displayed from stdout to stderr [#698](https://github.com/Scalingo/cli/pull/698)
```

Comment:
nitpick: I don't understand the `warning-data-stream`... Where does it come from? `update` is the name of the package you modified, sounds also good
```suggestion
* fix(update): change data stream on which warning is displayed from stdout to stderr [#698](https://github.com/Scalingo/cli/pull/698)
```